components/shell/sudosh2/sudosh2.p5m

```diff
@@ -0,0 +1,38 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Predrag Zečević
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# add explicit dependency to sudo
+depend fmri=pkg:/security/sudo type=require
+
+#
+dir path=/var/adm/sudosh mode=0733 group=root
```

Comment:
@alarcher 

this utility is wrapper to shell (both log and default shell are defined in /etc/sudosh.conf) which adds additional auditing for user account activity, so replacing user's shell with sudosh you (system administrator) can allow / deny commands and activity will be tracked and written to file under log directory.

So, log directory should be readable to root user only 0700 ?
Regards.